tests/integ/scala/test_table_suite.py

```diff
@@ -257,6 +261,10 @@ def test_table_with_semi_structured_types(session, semi_structured_table):
 
 
 # TODO: enable for local testing. Emulate data type
+@pytest.mark.skipif(
```

Comment:
I think this can be rewritten into calling df apis like the test case `table_with_time` :
```python
        session.create_dataframe(
            [[datetime.time(9, 15, 29)], [datetime.time(9, 15, 29, 999999)]],
            schema=StructType([StructField("time", TimeType())]),
        ).write.saveAsTable(table_name)
```